packages/core/test/integer.test.ts

```diff
@@ -412,66 +412,75 @@ describe('Integer', () => {
       (a, b, c) => a.multiply(b.subtract(c)).equals(a.multiply(b).subtract(a.multiply(c)))))
   })
 
+  test('Integer.add should have 0 as identity', () => {
+    fc.assert(fc.property(arbitraryInteger(), (a) => a.add(0).equals(a)))
+  })
+
+  test('Integer.subtract should have 0 as identity', () => {
+    fc.assert(fc.property(arbitraryInteger(), (a) => a.subtract(0).equals(a)))
+  })
+
+  test('Integer.multiply should have 0 as identity', () => {
+    fc.assert(fc.property(arbitraryInteger(), (a) => a.multiply(1).equals(a)))
+  })
+
+  test('Integer.div should have 0 as identity', () => {
```

Comment:
```suggestion
  test('Integer.div should have 1 as identity', () => {
```
please don't divide by 0 :wink:

Reply:
I'm creating a new math.